tests/test_html_output.py

```diff
@@ -201,14 +203,29 @@ def check_text_is_bold(element):
 
 
 def check_dark_mode(body_element, universum_log_element):
-    dark_mode_switch = TestElement.create(body_element.find_elements_by_xpath("./label")[0])
+    dark_mode_switch = TestElement.create(body_element.find_element_by_xpath("./label[@for='dark-checkbox']"))
     dark_mode_switch.click()
     assert universum_log_element.color == Color.WHITE
     assert universum_log_element.background_color == Color.BLACK
     dark_mode_switch.click()
     assert universum_log_element.color == Color.BLACK
 
 
+def check_timestamps(body_element, universum_log_element):
+    timestamp_element = universum_log_element.find_element_by_xpath("./*[@class='time']")
+    assert timestamp_element.is_displayed() is False
```

Comment:
IMHO, `assert not timestamp_element.is_displayed()` is closer to a human speech and thus more readable.